RELEASE-NOTES.md

```diff
@@ -13,6 +13,20 @@
   * Added `create` command for `image-repository`.
   * Added `set (property)`, `unset (property)`, `suspend` and `resume` commands for `compute-pool`.
   * Added `set (property)`, `unset (property)`,`upgrade` and `list-endpoints` commands for `service`.
+* Connections parameters are also supported by generic environment variables:
+  * `SNOWFLAKE_ACCOUNT`
+  * `SNOWFLAKE_USER`
+  * `SNOWFLAKE_PASSWORD`
+  * `SNOWFLAKE_DATABASE`
+  * `SNOWFLAKE_SCHEMA`
+  * `SNOWFLAKE_ROLE`
+  * `SNOWFLAKE_WAREHOUSE`
+  * `SNOWFLAKE_MFA_PASSCODE`
```

Comment:
I think we should remove mfa from here. I tested it in `test_connection_details_are_resolved_using_environment_variables` and didn't work as we expected.

Reply:
This won't be probably a common use case,  but there's probably nothing wrong in doing so